work/covidvu/pipeline/vujson.py

```diff
@@ -142,8 +142,12 @@ def makeUSRegionsVerbose(stateCodes, regionsUS = US_REGIONS):
 
 STATE_NAMES = {
                     'Washington, D.C.': 'Washington D.C.',
+                    'United States Virgin Islands': 'Virgin Islands',
+                    'Northern Mariana Islands': 'Marianas',
                 }
 
+STATE_NAMES_TO_DROP = ('US', 'Northern Mariana Islands', 'Recovered', 'Wuhan Evacuee')
```

Comment:
Let's talk later about the name homologizing function in `vudpatch`.  We do this already somewhere else.